classic_cp/latin/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Revise should not, in fact, be considered as a direct dependency of any project.

Reply:
done !